tokens/repository-map.json

```diff
@@ -17,6 +17,7 @@
     "circa": [
         "automated-interpretation-pipeline"
     ],
+    "common": [],
```

Comment:
Should we add `references` and `images` here?

Reply:
I have not created hail service accounts for the `common` dataset, so it's mostly just to ensure anything that's using a cached group members is actually picked up correctly - doesn't really do much else than that.